modifiers/fence_label.js

```diff
@@ -78,7 +75,7 @@ module.exports = (md, options) => {
         const className = optsObj.className || 'code-label';
 
         // Inject label and return
-        return `<div class="${md.utils.escapeHtml(className)}" title="${md.utils.escapeHtml(name)}">${md.utils.escapeHtml(name)}</div>
+        return `<div class="${md.utils.escapeHtml(className)}" title="${md.utils.escapeHtml(name) || ''}">${md.utils.escapeHtml(name) || ''}</div>
```

Comment:
What is the `|| ''` doing here? What is `md.utils.escapeHtml` returning that needs to be casted to an empty string?

Reply:
it returns null

Reply:
It that because `name` is null when passed in? Can we change this so that the `|| ''` is what gets passed into escapeHtml, rather than after -- just so that we're not encouraging a pattern of strings bypassing HTML escaping.